client/dashboard/sites/hosting-feature-activation-modal/data-center-form.tsx

```diff
@@ -0,0 +1,84 @@
+import { Button, Card, CardBody, SelectControl } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import InlineSupportLink from '../../components/inline-support-link';
+import { Text } from '../../components/text';
+
+const DataCenterOptions = [
```

Comment:
I think we should use the same data as 

https://github.com/Automattic/wp-calypso/blob/afc863359c80ba00c05e8f1632e472acb5204904/client/data/data-center/index.ts#L3-L10

Maybe we should put them in the new `api-core` package as constants 🤔 not sure